index.bs

```diff
@@ -692,14 +692,33 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. For each |authenticator| currently available on this platform, perform the following steps:
 
-    1. Let |credentialList| be a new [=list=].
-
-    1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a 
-        platform-specific procedure to determine which, if any, credentials in
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by matching with 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/id}}</code> and 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/type}}</code>, and set |credentialList| to
-        this filtered list.
+    <span class="note">Note: the procedure to determine the [=authenticators=] present on a platform is 
+        platform-specific.</span>
+
+    1. Let |credentialList| be a new [=ordered set=].
+
+    1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code>:
+        <dl class="switch">
+            : [=list/is not empty=]
+            :: [=set/For each=] credential |C| in <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code>:
+                1. If there is any credential |X| present on this |authenticator| that:
```

Comment:
Not for this PR: *X*s are actually not [credentials](https://w3c.github.io/webappsec-credential-management/#concept-credential) themselves either, they're [credential sources](https://w3c.github.io/webappsec-credential-management/#credential-source) (although that term is new and could change if you prefer a better option). This algorithm is about to use authenticatorGetAssertion to generate a credential from *X*.

This is a wide-ranging conflation in this spec, so calling *X*s "credentials" is fine for this PR.